### PR TITLE
Issue-70

### PR DESCRIPTION
**Pull request by cmoulliard**
- Add a HowTo section to explain how to play with primaza 
- Include a Table Of Content
- Explain the role of the deployed services
- See issue #70

### CODE REVIEW

**Review by aureamunoz — COMMENTED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -16,19 +25,35 @@ Today in Kubernetes, the exposure of secrets for connecting workloads to externa
...
 
-If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
+- Setup a kubernetes cluster
+- Deploy a backend service such as MySQL, PostgreSQL, etc on a k8s cluster (minikube, kind, etc)
+- Register the backend service using the UI, REST endpoint `http://localhost:8080/services/new`
+- Create a claim to bind the backend service to your application `http://localhost:8080/claims/new`
 
-## Running the application in dev mode
+If a match exists between the `claim` request and a `service` deployed (aka a kubernetes endpoint has been created for the service), then you should be able to
+get the secret after a few moment :-)
```
[the remark this answers is not in the document]
Comment (cmoulliard) [reply README.md:48]: > This is for the future, right? Currently we are not performing yet the secret injection into the pod

Yes but we should at least produce a secret ;-) Can we mount it using a Quarkus lib as we can do with https://github.com/spring-cloud/spring-cloud-kubernetes#secrets-propertysource ?

**Comment by cmoulliard** [comment]
Can we merge it ? @aureamunoz

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (aureamunoz) [reply README.md:48]: yes, ok. I don't know atm, I would need to figure it out.

**Comment by aureamunoz** [comment]
> Can we merge it ? @aureamunoz

Yes. Let's merge

**Review by aureamunoz — APPROVED** [review]

**Pull request merged**